Cargo.toml

```diff
@@ -7,6 +7,10 @@ edition = "2021"
 
 [dependencies]
 glam = { version = "0.22.0", features = ["serde"] }
+glutin = { version = "0.30.3", optional = true }
+raw-window-handle = "0.5.0"
+glutin-winit = { version = "0.2.1", optional = true }
+winit = { version = "0.27.5", optional = true }
```

Comment:
I see that `raw-window-handle` is not optional, is that intended? Logically it should be kept for the rendering feature.

Reply:
It isn't intended, sorry, but I wasn't sure under what feature should it be kept under.

Reply:
Maybe we should have a `rendering` feature that will be enabled by `wgpu` and `opengl`?

Reply:
And also `opengl` and `wgpu` would both require the `render` feature.